packages/middleware/src/types/common.ts

```diff
@@ -9,6 +10,7 @@ import {
 import { WithRequired } from "./index";
 import { ApiClient, ApiClientConfig, ApiClientFactory } from "./server";
 
+export type LoggerFactoryConfig = Parameters<typeof LoggerFactory.create>[1];
```

Comment:
Is this some kind of a weird inference of `LoggerOptions` type which is already exported?
Also, if this is meant to be a public interface then it should not be copied from the logger package (or infered in any way) but must be created manually, even if it will be a c/p, to not couple packages on the user interface level so we can avoid bc in the future if eg. logger options will change.